receptorctl/receptorctl/socket_interface.py

```diff
@@ -65,6 +66,23 @@ def readconfig(self, config, tlsclient):
                     self._insecureskipverify = key.get("insecureskipverify", self._insecureskipverify)
                     break
 
+    def check_version(self):
+        receptorVersion = self.simple_command('{"command":"status","requested_fields":["Version"]}')["Version"]
+        receptorctlVersion = pkg_resources.get_distribution('receptorctl').version
+        if receptorVersion != receptorctlVersion:
+            print("Warning: receptorctl and receptor are different versions, they may not be compatible")
+        else:
+            print("Versions match and should be compatible")
```

Comment:
I don't think it's necessary to print this in the case where the versions match.  It just distracts from whatever the real error is.